apps/ledger-live-desktop/src/renderer/analytics/segment.ts

```diff
@@ -56,9 +58,20 @@ const getFeatureFlagProperties = (): Record<string, boolean | string> => {
   try {
     if (!analyticsFeatureFlagMethod) return {};
     const ptxEarnFeatureFlag = analyticsFeatureFlagMethod("ptxEarn");
+    const fetchAdditionalCoins = analyticsFeatureFlagMethod("fetchAdditionalCoins");
+
+    const isBatch1Enabled =
+      !!fetchAdditionalCoins?.enabled && fetchAdditionalCoins?.params?.batch === 1;
+    const isBatch2Enabled =
+      !!fetchAdditionalCoins?.enabled && fetchAdditionalCoins?.params?.batch === 2;
+    const isBatch3Enabled =
+      !!fetchAdditionalCoins?.enabled && fetchAdditionalCoins?.params?.batch === 3;
```

Comment:
It seems very relative to swap....
Do we must move this logic to "useGetSwapTrackingProperties"
apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/utils/index.ts

Reply:
The ticket calls for these properties to be added to the segment identity on ledger-live open. 😃